model/logs/mobile-crashes.yaml

```diff
@@ -0,0 +1,47 @@
+groups:
+  - id: mobile-crash-event
+    prefix: hello
+    type: event
+    brief: >
+      This document defines attributes for crashes represented using Log Record Events.
+      `event.domain` is 'device' and `event.name` is 'crash'.
```

Comment:
`event.name` should allow something different than `crash`.
`crash` means the application exited, but this format should allow swallowed errors as well which is technically not a `crash` but rather an error or exception.

Reply:
The current thinking is that we will model a separate event type for non-fatal errors/exceptions. There is already an event type called `exception` in the spec but that's more for span events, that is, errors occuring in the context of a span, but we will need to model a similar event for a more generic usecase.